packages/dds/sequence/src/test/memory/.mocharc.js

```diff
@@ -14,7 +14,7 @@
     recursive: true,
     reporter: "../../../node_modules/@fluid-tools/benchmark/dist/MochaMemoryTestReporter.js", // Lerna hoists the external dependency on @fluid-tools/benchmark to the root
     reporterOptions: ["reportDir=.memoryTestsOutput/"],
-    require: ["node_modules/@fluidframework/mocha-test-setup"],
```

Comment:
Does it cause problem to leave `node_modules` there?   I am thinking for the scenario people wanting to run the test locally in the repo.  I assume removing it will break that scneario?

Reply:
Seems like I removed this by mistake! Will put it back! It doesn't cause a problem to the pipeline to keep it!

Reply:
Turns out you can't execute the tests locally without the `node_modules` in front of the package name! You get:

<img width="737" alt="image" src="https://user-images.githubusercontent.com/6777404/200444403-0a8ba7ba-be77-482d-9db2-ae3239bee517.png">

Reply:
Leaving it out was definitely a typo style mistake because the pipeline can't run it without node_modules either: https://dev.azure.com/fluidframework/internal/_build/results?buildId=106695&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&t=f792184b-eadf-5696-8c87-1b052e836e1a&l=203

Reply:
The pipeline is complaining about a slightly different thing, though (the reporter file, not the mocha-test-setup package), and that's related to one of the paths in because as the comment in `.mocharc.js` says, when running

Reply:
@alexvy86 yup you're right about the pipeline portion -- it's the `reporter` field with the three levels up that needs to be cleaned up! This is probably what I mean to remove originally!

However, the `require` field does need to have `node_modules` in front of it because it fails to run locally otherwise, so removing this was a mistake!

Reply:
I created a [task](https://dev.azure.com/fluidframework/internal/_workitems/edit/2528) for myself to document all the complexities we've encountered (and hopefully, now fully understood 😄) related to how/why to specify paths when invoking mocha (and node in general, since `-r/--require` is not specific to mocha).